config/README.md

```diff
@@ -0,0 +1,15 @@
+# GOV.UK Chat Evaluation
+
+## Config
+
+This directory is for storing config files used for evaluations. 
+
+The ones in the default directory are committed to the repository whereas other files within it, or within other directories, will only be available on your local system.
+
+Make changes to the default files if you want to affect how someone typically runs an evaluation or for a common evaluation scenario. For your own specific testing, use other locations.
```

Comment:
If one uses "other locations", how should one pass the ad-hoc config filepath to the `uv run govuk_chat_evaluation <TASK>` command? 

I guess it is by passing a value to the `--config_path` argument?

Reply:
Yup thats correct. I've added some detail to this config file to include that.